blog/2024-02-22/index.md

```diff
@@ -0,0 +1,47 @@
+
+## What are brand colors?
+
+App brand colors are an easy and effective way for app developers to give their listing a bit more personality in app stores. In combination with the app icon and name, they allow setting a tone for the app without requiring a lot of extra work, unlike e.g. creating and maintaining additional image assets.
+
+![GNOME Software Explore view with brand colors on app banners](software-explore.png)
+
+## Why now?
+
+This idea was first implemented in elementary's Appcenter, and later standardized as part of the appstream specification.
+
+While it has been in appstream [for a few years](https://github.com/ximion/appstream/issues/187), it was unfortunately not possible for Flathub's backend to pick it up until the port to libappstream a few weeks ago. This is why many apps are still not providing this metadata as we have not been able to make of it until now.
+
+![elementary Appcenter with brand colors on app banners](appcenter.png)
+
+Now that we can finally pick up these colors from appstream metainfo files we want to make use of them, and they are essential for the new banners.
+
+## Adding brand colors
+
+Apps are expected to provide two different brand colors for light and dark. [Here's an example](https://github.com/pika-backup/pika-backup/blob/8f2f04a1a27d2b04db48f7dbf26577009ff39be3/data/app.metainfo.xml.in#L44) of an appstream file in the wild including brand colors.
+
+This is the snippet you need to include in your metainfo file:
+
+```
+<branding>
+  <color type="primary" scheme_preference="light">#faa298</color>
+  <color type="primary" scheme_preference="dark">#7f2c22</color>
+</branding>
+```
+
+In choosing the colors, try to make sure the colors work well in their respective context (e.g. don't use a light yellow for dark mode), and look good as a background behind the app icon (e.g. avoid using exactly the same color to maintain contrast). In most cases it's recommended to pick a lighter (or darker, for dark mode) version of one of the main colors from the icon. Alternatively you can also go with a complementary color that goes well with the icon's colors.
+
+![Three examples of good/bad brand colors](color-examples.png)
+
+## What's next?
+
+Today we've updated the metadata quality guidelines with [a new section on app brand colors](https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/quality-guidelines/#brand-colors). The metadata validation system on the Flathub website now also has an additional check for whether the brand colors are provided, and they're required to fully pass metadata quality review.
```

Comment:
MetaInfo; also, the last sentence was kind of saying the same thing twice? Maybe simplify something like this:

```suggestion
Today we've updated the MetaInfo quality guidelines with [a new section on app brand colors](https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/quality-guidelines/#brand-colors). Going forward, brand colors will required as part of the MetaInfo quality review.
```

Reply:
Link should probably be relative

Reply:
I don't think the last sentence is complete?

Reply:
What would a relative link look like for this? Got an example?

Reply:
`/docs/for-app-authors/metainfo-guidelines/quality-guidelines/#brand-colors`

Should work, depending on which name we come up with for the headline